api/src/main/java/org/openmrs/Location.java

```diff
@@ -33,57 +58,92 @@
 	
 	// Fields
 	
+	@Id
+	@Column(name = "location_id", nullable = false)
+	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "location_id_gen")
+	@SequenceGenerator(name = "location_id_gen", sequenceName = "location_location_id_seq")
 	private Integer locationId;
 	
+	@Column(name = "address1")
 	private String address1;
 	
+	@Column(name = "address2")
 	private String address2;
 	
+	@Column(name = "city_village")
 	private String cityVillage;
 	
+	@Column(name = "state_province")
 	private String stateProvince;
 	
+	@Column(name = "country", length = 50)
 	private String country;
 	
+	@Column(name = "postal_code", length = 50)
 	private String postalCode;
 	
+	@Column(name = "latitude", length = 50)
 	private String latitude;
 	
+	@Column(name = "longitude", length = 50)
 	private String longitude;
 	
+	@Column(name = "county_district")
 	private String countyDistrict;
-	
+
+	@Column(name = "address3")
 	private String address3;
-	
+
+	@Column(name = "address4")
 	private String address4;
-	
+
```

Comment:
you could run mvn clean package before pushing the code to avoid the unnecessary code formats @Ruhanga  how does this look to you?